Sources/XCHammer/XCBuildSettings.swift

```diff
@@ -161,6 +161,8 @@ struct XCBuildSettings: Encodable {
     var moduleMapFile: First<String>?
     // Disable Xcode derived headermaps, be explicit to avoid divergence
     var useHeaderMap: First<String>? = First("NO")
+    var swiftVersion: First<String>?
```

Comment:
@rahul-malik is this the correct data type in Xcode?

Reply:
yep it is!